mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/json/IndexableMessageTest.java

```diff
@@ -61,7 +61,7 @@
     public static final MessageUid MESSAGE_UID = MessageUid.of(154);
 
     @ClassRule
-    public static TikaContainer tika = new TikaContainer();
+    public static TikaContainerStaticRule tika = TikaContainerStaticRule.rule;
```

Comment:
how do you stop docker tika after tests?

Reply:
It's testcontainers resource reaper which ensures that.

But you are right. I need at least a comment.